build.gradle

```
@@ -56,13 +56,13 @@ ext {
     MCP_VERSION = '20220404.173914'
     SPI_VERSION = '4.0.15-4.x'
     ACCESSTRANSFORMERS_VERSION = '8.0.4'
-    COREMODS_VERSION = '5.0.1'
-    EVENTBUS_VERSION = '5.0.3'
+    COREMODS_VERSION = '5.1.2'
+    EVENTBUS_VERSION = '5.0.7'
```

```suggestion
    EVENTBUS_VERSION = '5.0.3'
```
https://maven.minecraftforge.net/net/minecraftforge/eventbus/5.0.7/eventbus-5.0.7-changelog.txt
No changes for consumers going from 5.0.3 to 5.0.7, so please leave it as-is so that the cached version from old installs can still be used to keep updates fast on slow connections.